versions-maven-plugin/src/it/it-display-dependency-updates-issue-973-versionless/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>localhost</groupId>
+  <artifactId>it-display-dependency-updates-002</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+  <name>display-dependency-updates-from-plugins</name>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>localhost</groupId>
+        <artifactId>dummy-api</artifactId>
+        <version>1.1</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>localhost</groupId>
+          <artifactId>dummy-maven-plugin</artifactId>
+          <dependencies>
+            <dependency>
+              <groupId>localhost</groupId>
+              <artifactId>dummy-api</artifactId>
+            </dependency>
+          </dependencies>
```

Comment:
I think that such example is wrong ... it is not possible to execute such project
version in plugin/dependencies must be set and it is not resolved from project dependencies/depndcyManagemnt

Reply:
This example is the only case I've found that reproduces the issue. Yes, it does work with Maven.

Reply:
please look at my commit and build status

Reply:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'build.plugins.plugin[org.apache.maven.plugins:maven-clean-plugin].dependencies.dependency.version' for localhost:dummy-api:jar is missing. @ line 27, column 25
```

Reply:
Strange. I don't understand why it worked with my commit then. With my pom.xml, the issue was being hit and the fix did work. 

So now it remains to be found how we could possibly reproduce the issue.

Reply:
In your commit plugin was not executed - only versions-plugin try to analyze project